Snoop.InjectorLauncher/ProcessWrapper.cs

```diff
@@ -92,9 +92,20 @@ private static string GetSupportedTargetFramework(Process process)
         {
 #if DEBUG
             Injector.LogMessage(module.szExePath);
-            var fileVersionInfo = FileVersionInfo.GetVersionInfo(module.szExePath);
-            Injector.LogMessage($"File: {fileVersionInfo.FileVersion}");
-            Injector.LogMessage($"Prod: {fileVersionInfo.ProductVersion}");
+            try
```

Comment:
It would be better to just check if the file exists instead of catching an exception.